Makefile

```diff
@@ -27,9 +27,10 @@ test:
 .PHONY: integration-test
 # Run integration tests
 integration-test: cli-fast
```

Comment:
@nfnt I had to make these changes in order to run the tests locally. It's super important for users to be able to run the tests locally or they won't write new ones.

Reply:
💯 Thanks for catching that!